tests/unit/changes_tests.py

```diff
@@ -245,7 +245,6 @@ def test_get_feed_descending(self):
                         self.assertTrue(current < last)
                     except AssertionError:
                         self.assertEqual(current, last)
-                        self.assertTrue(len(change['seq']) > len(last_seq))
```

Comment:
Please can you update the test comment block above to remove the reference to this assertion too.

Reply:
Done in 8d968f4